javascript-source/core/whisper.js

```diff
@@ -85,7 +85,7 @@
             return;
         }
 
-        if (message.startsWith('!') && $.isMod(sender) && $.users.includes(sender)) {
+        if (message.startsWith('!') && $.isMod(sender) && $.userExists(sender)) {
```

Comment:
The function `$.userExists` basically calls the same method as `includes`, but with Java code with is what is needed in this case. I agree with this change.